f5lbaasdriver/test/tempest/tests/api/base.py

```diff
@@ -30,7 +35,146 @@
 class F5BaseTestCase(base.BaseTestCase):
     """This class picks non-admin credentials and run the tempest tests."""
 
+    config_file = "/tmp/tempest.bigip.cfg"
     _lbs_to_delete = []
+    ssh_cmd = \
+        str("ssh -F /home/ubuntu/testenv_symbols/testenv_ssh_config"
+            " openstack_bigip")
```

Comment:
the `openstack_bigip` has to be set along with the config file on your system... also there is the requirement that you setup an `ssh-key` via `ssh-keygen` that is shared between you and your bigip for this to work.  This is more of an 'environment' on what this might look like...